backend/core/src/app.module.ts

```diff
@@ -41,7 +41,18 @@ import { CatchAllFilter } from "./shared/filters/catch-all-filter"
 
 export function applicationSetup(app: INestApplication) {
   const { httpAdapter } = app.get(HttpAdapterHost)
-  app.enableCors()
+  const allowList = process.env.CORS_ORIGINS
```

Comment:
might be a good idea just in case we miss a config file configuration in one of the environments we should change this line to 
`const allowList = process.env.CORES_ORIGINS || []`